etl/steps/data/garden/energy/2023-12-12/photovoltaic_cost_and_capacity.py

```diff
@@ -144,6 +138,10 @@ def run(dest_dir: str) -> None:
     # Create a table of cumulative solar photovoltaic capacity, by combining Nemet (2009) and IRENA data.
     cumulative_capacity = prepare_capacity_data(tb_nemet=tb_nemet, tb_irena_capacity=tb_irena_capacity)
 
+    # Sanity check.
+    error = "IRENA data has changed, prices may need to be deflated to the latest year."
+    assert tb_irena_cost["year"].max() == LATEST_YEAR, error
```

Comment:
I guess you can do something similar for your own calculations - just to be safe as you are adding the numbers manually each time you are doing an update